huaweicloud/services/vpc/data_source_huaweicloud_vpc_ids.go

```diff
@@ -1,15 +1,17 @@
 package vpc
 
 import (
+	"context"
+
 	"github.com/chnsz/golangsdk/openstack/networking/v1/vpcs"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
```

Comment:
```suggestion
	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"

	"github.com/chnsz/golangsdk/openstack/networking/v1/vpcs"
	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
```
The imported packages are sorted by stability.